templates/search_page.html

```diff
@@ -136,9 +136,22 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
 
                     <div class="flex-fill">
                         <input class="form-control border-0" type="search" id="location" placeholder="Search"
-                            aria-label="Search" name="location" value="Los Angeles, CA, USA">
+                            aria-label="Search" name="location" value="" onchange="fetchLoc(this)">
+                        <script>
+                            try {
```

Comment:
Why do we need try catch block here?

Reply:
to catch an error if the `loc` is not defined or cannot be got